src/app/(root)/product/components/index.tsx

```diff
@@ -0,0 +1,114 @@
+"use client"
+
+import * as React from "react"
+import { AlertDialog } from "@mijn-ui/components/alert-dialog"
+import { Table } from "@mijn-ui/components/table"
+import SelectionMenu from "@/components/menu/selection-menu"
+import TablePaginator from "@/components/table/table-paginator"
+import { useProductTable } from "../hooks/use-product-table"
+import DeleteModalContent from "./delete-modal-content"
+import TableActions from "./table-actions"
+import TableBody from "./table-body"
+import TableErrorMessage from "./table-error-message"
+import TableHeader from "./table-header"
+
+/* -------------------------------------------------------------------------- */
+
+const ItemsPerPageArray = ["10", "20", "30"]
+
+const ProductTable: React.FC = () => {
+  const {
+    itemsPerPage,
+    setItemsPerPage,
+    currentPage,
+    setCurrentPage,
+    searchInput,
+    selectedProductsId,
+    isAllProductSelected,
+    products,
+    isLoading,
+    isError,
+    handleCheck,
+    handleSelectAll,
+    handleSort,
+    handleInputChange,
+    isDeleteDialogOpen,
+    setIsDeleteDialogOpen,
+    handleDelete,
+    isDeleteLoading,
+  } = useProductTable()
+
+  return (
+    <>
+      <AlertDialog
+        open={isDeleteDialogOpen}
+        onOpenChange={setIsDeleteDialogOpen}
+      >
+        <TableActions
+          searchInput={searchInput}
+          handleInputChange={handleInputChange}
+          totalSelectedUsers={selectedProductsId.length}
+        />
+
+        <div className="relative w-full h-full max-h-max overflow-auto bg-surface">
```

Comment:
```suggestion
        <div className="custom_scroll_bar relative w-full h-full max-h-max overflow-auto bg-surface">
```

Add custom_scroll_bar class to make the table scrollbar looks great,